### PR TITLE
CFE-636 Test that methods can be called by expanding classic arrays

### PR DESCRIPTION
**Pull request by nickanderson**
(cherry picked from commit b61e3fe853f10409152bb1424b6748ffa3266734)

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a pull request! Maybe @olehermanse can review this?

**Comment by olehermanse** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/1882/)

https://ci.cfengine.com/job/pr-pipeline/1882/

File: tests/acceptance/21_methods/call_methods_using_array_expansion.cf [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#######################################################
+#
+# Redmine 6369 - make sure that arrays don't accumulate
+#
```
Comment (craigcomstock) [inline tests/acceptance/21_methods/call_methods_using_array_expansion.cf]: CFE-636 instead of Redmine 6369?

Comment (olehermanse) [reply tests/acceptance/21_methods/call_methods_using_array_expansion.cf]: Cherry-pick

**Pull request merged**